components/usage/pkg/stripe/stripe.go

```diff
@@ -146,15 +149,15 @@ func (c *Client) updateUsageForCustomer(ctx context.Context, customer *stripe.Cu
 			Context: ctx,
 		},
 		SubscriptionItem: stripe.String(subscriptionItemId),
-		Quantity:         stripe.Int64(summary.Credits),
+		Quantity:         stripe.Int64(credits),
 	})
 	if err != nil {
 		return nil, fmt.Errorf("failed to register usage for customer %q on subscription item %s", customer.Name, subscriptionItemId)
 	}
 
 	return &UsageRecord{
 		SubscriptionItemID: subscriptionItemId,
-		Quantity:           summary.Credits,
+		Quantity:           credits,
```

Comment:
nit: do we use this struct anywhere? the name `UsageRecord` might be a bit confusing given we have records called `Usage`.

Reply:
Fair. It was mostly to return something from method but we can easily return a tuple instead.